test/renaissance/auctions/auction_test.exs

```diff
@@ -0,0 +1,139 @@
+defmodule Renaissance.Test.AuctionTest do
+  use Renaissance.DataCase
+
+  require Ecto.Query
+  alias Renaissance.{Auction}
+
+  describe "user" do
+    test "requires title" do
+      {:ok, datetime, 0} = DateTime.from_iso8601("2019-04-21T12:00:00+0000")
+
+      changeset =
+        Auction.changeset(
+          %Auction{},
+          %{
+            title: "",
+            description: "Test description",
+            user_id: 1,
+            price: 100,
+            end_date: datetime
+          }
+        )
+
+      refute changeset.valid?
+      assert %{title: ["can't be blank"]} = errors_on(changeset)
+    end
+
+    test "requires description" do
+      {:ok, datetime, 0} = DateTime.from_iso8601("2019-04-21T12:00:00+0000")
+
+      changeset =
+        Auction.changeset(
+          %Auction{},
+          %{
+            title: "Test title",
+            description: "",
+            user_id: 1,
+            price: 100,
+            end_date: datetime
+          }
+        )
+
+      refute changeset.valid?
+      assert %{description: ["can't be blank"]} = errors_on(changeset)
+    end
+
+    test "requires user id" do
```

Comment:
It would be nice to have a test for an invalid user_id (one that isn't a valid foreign key based on the `references(:users)` foreign key constraint.